src/main/java/com/pngencoder/PngEncoderPhysicalPixelDimensions.java

```diff
@@ -0,0 +1,74 @@
+package com.pngencoder;
+
+/**
+ * Represents PNG physical piel dimensions
+ *
+ * Use on of the static methods to create physical pixel dimensions based
+ * on pixels per meter, dots per inch or a unit-less aspect ratio.
+ *
+ * @see <a href="https://www.w3.org/TR/PNG/#11pHYs">https://www.w3.org/TR/PNG/#11pHYs</a>
+ */
+public class PngEncoderPhysicalPixelDimensions {
+
+    public enum Unit {
+        UKNOWN((byte) 0),
+        METER((byte) 1);
+
+        private final byte value;
+
+        Unit(byte value) {
+            this.value = value;
+        }
+
+        public byte getValue() {
+            return value;
+        }
+    }
+
+    private static final float INCHES_PER_METER = 100 / 2.54f;
+
+    private final int pixelsPerUnitX;
+    private final int pixelsPerUnitY;
+    private final Unit unit;
+
+    private PngEncoderPhysicalPixelDimensions(int pixelsPerUnitX, int pixelsPerUnitY, Unit unit) {
+        this.pixelsPerUnitX = pixelsPerUnitX;
+        this.pixelsPerUnitY = pixelsPerUnitY;
+        this.unit = unit;
+    }
+
+    public static PngEncoderPhysicalPixelDimensions pixelsPerMeter(int pixelsPerMeterX, int pixelsPerMeterY) {
+        return new PngEncoderPhysicalPixelDimensions(pixelsPerMeterX, pixelsPerMeterY, Unit.METER);
+    }
+
+    public static PngEncoderPhysicalPixelDimensions pixelsPerMeter(int pixelsPerMeter) {
+        return pixelsPerMeter(pixelsPerMeter, pixelsPerMeter);
+    }
+
+    public static PngEncoderPhysicalPixelDimensions dotsPerInch(int dotsPerInchX, int dotsPerInchY) {
```

Comment:
Maybe a javadoc explaining the lossy behavior of the rounding here?

Reply:
I've added some Javadoc to the class - perhaps we should add it in more places, if we want to make it easy to use this library.